NuGet.Config

```diff
@@ -16,7 +16,7 @@
     <add key="myget.org roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="myget.org roslyn tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
-    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn-for-vs-for-mac/api/v3/index.json" />
+    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
To confirm: this change is _not_ necessary for this to build right, because you're not depending on any _new_ Roslyn internals?

(I'm happy if you're moving back to mainline, but if there's a new dependency then something is very fishy here.)

Reply:
Correct, this isn't required for this; I'm just trying to cleanup our dependencies.